var/spack/repos/builtin/packages/py-locket/package.py

```diff
@@ -12,7 +12,10 @@ class PyLocket(PythonPackage):
     homepage = "https://github.com/mwilliamson/locket.py"
     pypi = "locket/locket-0.2.0.tar.gz"
 
+    version("1.0.0", sha256="5c0d4c052a8bbbf750e056a8e65ccd309086f4f0f18a2eac306a8dfa4112a632")
     version("0.2.0", sha256="1fee63c1153db602b50154684f5725564e63a0f6d09366a1cb13dffcec179fb4")
 
     # pip silently replaces distutils with setuptools
     depends_on("py-setuptools", type="build")
+    depends_on("python@2.7:", type="test", when="@1:")
+    conflicts("python@3:3.3", when="@1:")
```

Comment:
You could say:
```python
depends_on("python@2.7:2,3.4:", type=("build", "run"))
```
or you could just remove both of these lines since Spack no longer supports Python 3.6 and older anyway